### PR TITLE
=doc Improve readme a bit, more links, and sections

### PR DESCRIPTION
**Pull request by ktoso**
Minor improvements of README.md, to link to our sites more prominently etc.
Content is mostly the same as was previously.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-# Akka
+Akka
+====
```
Comment (ktoso) [inline README.md]: The intro is a bit old, could use a rewrite.

**Comment by ktoso** [comment]
cc @patriknw

**Comment by ktoso** [comment]
(Preview https://github.com/ktoso/akka/blob/9c844f2830366835022434a78fd62398715ad48f/README.md )

**Comment by ktoso** [comment]
Refs https://github.com/akka/akka/issues/20862#issuecomment-230487008

File: README.md [hunk 2]
```diff
@@ -10,10 +12,38 @@ For resilience we adopt the "Let it crash" model which the telecom industry has
 
 Actors also provide the abstraction for transparent distribution and the basis for truly scalable and fault-tolerant applications.
 
-Akka is Open Source and available under the Apache 2 License.
+Learn more at [akka.io](http://akka.io/).
```
Comment (patriknw) [inline README.md]: should we also add direct links to the (html) docs for `current` Java and Scala docs?

**Comment by patriknw** [comment]
LGTM

File: README.md (hunk 2 above)
Comment (ktoso) [inline README.md]: Good idea, I'll add

**Comment by ktoso** [comment]
Thanks, addressed comments and merged

**Pull request merged**